todo_app/accounts/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import path, include
+from .api import RegisterAPI, LoginAPI, UserAPI, FindUserAPI
+from knox import views as knox_views
+
+urlpatterns = [
+    path('api/auth', include('knox.urls')),
+    path('api/auth/register', RegisterAPI.as_view(), name='register'),
+    path('api/auth/login', LoginAPI.as_view(), name='login'),
+    path('api/auth/user', UserAPI.as_view(), name='get_user'),
+    path('api/auth/logout', knox_views.LogoutView.as_view(), name='knox_logout'),
```

Comment:
What is the purpose of the `knox_` stuff? Your login and logout functionality should probably be housed together.